### ISSUE TITLE
Cache cases in redis for performance purposes

### ISSUE THREAD

**Issue by trystant** [EBWiki/EBWiki#2696]
Currently, cases are being loaded from the db every time, which is a poor use of both the database & not use of the Redis.cache.  Loading the `/maps` page multiple times in dev shows these results:
```
Completed 200 OK in 14257ms (Views: 6209.1ms | ActiveRecord: 2059.4ms)
Completed 200 OK in 19051ms (Views: 7622.5ms | ActiveRecord: 3274.5ms)
Completed 200 OK in 15672ms (Views: 6311.3ms | ActiveRecord: 2530.8ms)
Completed 200 OK in 17250ms (Views: 7236.9ms | ActiveRecord: 2535.0ms)
Completed 200 OK in 17286ms (Views: 7506.3ms | ActiveRecord: 2431.9ms)
```

**Comment by trystant** [comment]
I am going to leave the Rubocop errors for another ticket. For this one, I believe that this is ready for a PR if one doesn't already exist

**Comment by rlgreen91** [comment]
Merged in #2703
